docs/codacy-api/examples/creating-project-api-tokens-programmatically.md

```diff
@@ -9,6 +9,35 @@ To create new [project API tokens](../api-tokens.md) for your Codacy repositorie
 
 For example, if you're [setting up coverage](../../coverage-reporter/index.md) for all your repositories and prefer not to use a single account API token that grants the same permissions as an administrator, you need to create an individual project API token for each repository.
 
+## Example: Creating a project API token for a single repository
+
+This example creates new project API token for a repository and outputs the new token string.
+
+The example script:
+
+1.  Defines the [account API token](../api-tokens.md#account-api-tokens) used to authenticate on the Codacy API, the Git provider, the organization name, and the repository name passed as an argument to the script.
+1.  Calls the endpoint [createRepositoryApiToken](https://app.codacy.com/api/api-docs#createrepositoryapitoken) to create a new project API token and uses jq to obtain only the created token string.
```

Comment:
One minor suggestion: we do link to the `jq` repo in other examples.

```suggestion
1.  Calls the endpoint [createRepositoryApiToken](https://app.codacy.com/api/api-docs#createrepositoryapitoken) to create a new project API token and uses [jq](https://github.com/stedolan/jq) to obtain only the created token string.
```

Reply:
Thanks, indeed it's best to link to jq as not all environments will have it installed by default. :+1: